pkg/lwip/Makefile

```diff
@@ -1,5 +1,5 @@
 PKG_NAME=lwip
-PKG_URL=git://git.savannah.nongnu.org/lwip.git
+PKG_URL=git://git.savannah.nongnu.org/git/lwip.git
```

Comment:
unrelated? Otherwise: please explain.

Reply:
Was trying to get a TCP/IP stack before someone pointed out gnrc_tcp. In attempting to use pkg lwip kept getting a failure to resolve the github repo. On investigating found the repo URL was missing the "/git" so I updated it to test with pkg lwip. I will remove that change in my next commit.